mixer/cmd/build.go

```diff
@@ -662,6 +683,9 @@ func init() {
 	_ = buildBundlesCmd.Flags().MarkHidden("new-chroots")
 	_ = buildBundlesCmd.Flags().MarkDeprecated("new-chroots", "new functionality is now the standard behavior, this flag is obsolete and no longer used")
 
+	buildValidateCmd.Flags().IntVar(&buildFlags.to, "to", 0, "To version")
+	buildValidateCmd.Flags().IntVar(&buildFlags.from, "from", 0, "From version")
```

Comment:
It would be nice to have a more descriptive explanation of what "To"and "From" are since this is a help message.